### ISSUE TITLE
CentOS 6 parted.sh just spins endlessly

### ISSUE THREAD

**Issue by jpforte** [EugenMayer/parted-auto-resize#2]
it appears that stdin to part causes it to hang.

when I execute the command ./resize.sh /dev/sdb 1 the command hangs.

When I do 

> printf %sn 'unit MB print list' | parted 

The cursor runs back and forth as if caught in a loop

Killing from another console kills it.

**Comment by EugenMayer** [comment]
feel free to patch it and provide a PR

**Comment by AndrewSav** [comment]
Yeah, nah, it's not going to work on centos 6. `parted` on centos 6 is of version 2.1 and it does not even have resizepart command.

@EugenMayer could you please clearly specify in readme what os flavours versions and what parted version you tested this with. Many thanks.

**Comment by EugenMayer** [comment]
did so, but be aware, you could easily create a patch to have it compatible 

"HINT: this script is compatible with parted v4+ OOTB, if you have parted 3, you need to change parted resizepart to parted resize" see https://serverfault.com/a/870595/281162

so if you would like to contribute and test this, would be lovely to have a PR about that

**Comment by andrewsav-bt** [comment]
@EugenMayer The [latest parted is 3.2](https://ftp.gnu.org/gnu/parted/) or are you talking about some different parted?

**Comment by EugenMayer** [comment]
You right, 4.x was just completely confusion - corrected the minimal version (3.x)
Thanks for pointing me at that
